coral/proxy/README.md

```diff
@@ -0,0 +1,126 @@
+
+**Use `pnpm dev:start` when:**
+
+- you run the docker container for the first time
+- there are changes in `/core` or `/cluster-api`, where you want a new build to run against the most current version
+
+### ➡️ `pnpm dev:restart`
+
+This is the alias for `pnpm dev --mode=restart`.
+
+Checks if Klaw core and Klaw cluster-api are running in the right ports in your docker container. If they are not, it will trigger a new deploy, which will restart your container.
+
+**Use `pnpm dev:restart` when:**
+
+- you have a current version Klaw already in your container that you only want to restart
+
+### ➡️ `pnpm:[start|restart]:testEnv`
+
+This is the alias for `pnpm dev --mode=[start|restart] --testEnv=true`.
+
+Runs the `pnpm dev` command with the mode flag start or restart and an additional flag "testEnv". When `--testEnv=true` is set, we additionally check and, if necessary, setup a sandbox environment for klaw (zookeeper, kafka, schema-registry). Note: The files for the sandbox environment are large.
+
+**Use `pnpm:[start|restart]:testEnv` when:**
+
+- you want to go through whole workflows, like for example requesting and approving a topic
+
+### ➡️ `pnpm dev:stop`
+
+This is the alias for `pnpm dev --mode=stop`.
+
+This will stop all our docker containers. You can also do that in the Docker Dashboard if you're using a desktop app.
+
+Note: You don't need to stop containers before you run `pnpm dev:start` for a new build.
+
+**Use `pnpm dev:stop` when:**
+
+You don't need the api anymore and want to stop the containers, and you probably don't need a new state next time you want to work with them again. You can start them up again next time with `pnpm dev:restart`.
+
+### ➡️ `pnpm dev:destroy`
+
+This is the alias for `pnpm dev --mode=destroy`.
+
+This will stop and teardown all our docker containers. You can also do that in the Docker Dashboard if you're using a desktop app.
+
+Note: You don't need to destroy containers before you run `pnpm dev:start` for a new build.
+
+**Use `pnpm dev:destroy` when:**
+
+- You don't need the api anymore and want to stop the containers, and you probably will need to use a new build next time anyway.
+
+### ➡️ `pnpm setup`
+
+This is the alias for `pnpm dev --mode=all --testEnv=true`.
+
+This will build Klaw, the cluster-api and a sandbox environment for klaw (container for zookeeper, kafka, schema-registry). You usually won't need that script, it is a convenient shortcut for the first setup.
+
+### ➡️ additional `--verbose` flag
+
+Adding this flag to any of the mentioned commands will run the proxy in "verbose" mode. We will print **all** requests that are handled in the terminal. This is very noisy, so not enabled by default. It can be a great way to debug in case something is not working like expecting.
+
+## How it works
+
+### These scripts are used by other scripts
+
+- `_internal_use_proxy` starts the proxy. It will automatically restart on changes.
```

Comment:
This may be pedantic (or ignorant, if I misunderstand what is happening), but it's exactly the kind of thing that trips me up when reading that kind of documentation ^^

Changes to what? :o

Because I see two things in the server setup:
- we forward the HMR changes to the proxy server with these lines: https://github.com/aiven/klaw/pull/1482/files#diff-9a18be08938a76ab578699251ed0351a671667053349523b0fdec308d4d7587fR67-R74
- we start the server with `nodemon` which will restrat the server on changes done to the directory. But I don't believe we ever would do any changes to the `proxy` directory? Except when we are actually doing changes to the proxy haha! 

But this "on changes" is ambiguous, and I think you mean "on changes to coral", in which case it doesn;t "restart" but "forward the changes". 

Of course, if both are required for HMR to work in this setup, then forget what I said. But also probably should be made clear ^^

Reply:
Good point 😬  I guess this is not needed as information, since it's only relevant when working on the proxy itself, hu? 🙈

Reply:
Yes, maybe the second sentence is superfluous and it can just be " `_internal_use_proxy` starts the proxy. "

Reply:
Removed it!